src/request_executor.cc

```diff
@@ -365,6 +365,36 @@ RequestExecutor::Infer(
           infer_request->Trace().triton_trace_, &trace));
     }
 
+    const std::string& param_str = infer_request->Parameters();
+    triton::common::TritonJson::Value param;
+    THROW_IF_TRITON_ERROR(param.Parse(param_str.c_str(), param_str.length()));
+    std::vector<std::string> param_keys;
+    THROW_IF_TRITON_ERROR(param.Members(&param_keys));
+    for (const auto& key : param_keys) {
+      triton::common::TritonJson::Value value;
+      if (!param.Find(key.c_str(), &value)) {
+        throw PythonBackendException("Unexpected missing key on parameters");
+      }
+      if (value.IsString()) {
+        std::string string_value;
+        THROW_IF_TRITON_ERROR(value.AsString(&string_value));
+        THROW_IF_TRITON_ERROR(TRITONSERVER_InferenceRequestSetStringParameter(
+            irequest, key.c_str(), string_value.c_str()));
+      } else if (value.IsInt()) {
+        int64_t int_value = 0;
+        THROW_IF_TRITON_ERROR(value.AsInt(&int_value));
+        THROW_IF_TRITON_ERROR(TRITONSERVER_InferenceRequestSetIntParameter(
+            irequest, key.c_str(), int_value));
+      } else if (value.IsBool()) {
+        bool bool_value = false;
+        THROW_IF_TRITON_ERROR(value.AsBool(&bool_value));
+        THROW_IF_TRITON_ERROR(TRITONSERVER_InferenceRequestSetBoolParameter(
```

Comment:
Basic question: what was the reason of not supporting floats?

Reply:
It is because the core only has APIs for string, int and bool.
https://github.com/triton-inference-server/core/blob/main/src/tritonserver.cc#L1974-L2000

Reply:
I think the user can always use a string to represent floats.

Reply:
> It is because the core only has APIs for string, int and bool.
I saw that, hoped there might be more details behind this choice
> I think the user can always use a string to represent floats.
agree, if they need a WAR, this might be a potential solution

Reply:
>>> Basic question: what was the reason of not supporting floats?

>> It is because the core only has APIs for string, int and bool.
https://github.com/triton-inference-server/core/blob/main/src/tritonserver.cc#L1974-L2000

> I saw that, hoped there might be more details behind this choice

I don't know more details behind it. Maybe @tanmayv25 or @Tabrizian might know more details on the data types behind parameters?

Reply:
I think it was just not requested. If someone asks for it we can add support for this.

Reply:
CC @fpetrini15 , I believe this PR related to your ticket: https://jirasw.nvidia.com/browse/DLIS-5626